.github/workflows/build_wheels_linux.yml

```diff
@@ -75,11 +75,12 @@ jobs:
       REF: ${{ inputs.ref }}
       CU_VERSION: ${{ matrix.desired_cuda }}
       UPLOAD_TO_BASE_BUCKET: ${{ matrix.upload_to_base_bucket }}
+      GPU_FLAG: "${{ matrix.gpu_arch_type == 'cuda' && '--gpus all' || '' }}"
     name: ${{ matrix.build_name }}
     runs-on: ${{ matrix.validation_runner }}
     container:
       image: ${{ matrix.container_image }}
-      options: ${{ matrix.gpu_arch_type == 'cuda' && '--gpus all' || ' ' }}
+      options: ${{ env.GPU_FLAG }}
```

Comment:
You will need to delete both of these lines for a clean revert, since the original change was implemented intwo separate PR's